server/server/Models/DatabaseContext.cs

```diff
@@ -22,7 +22,7 @@ public class DatabaseContext : DbContext
 
         protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
-            optionsBuilder.UseMySql("Server=localhost;Database=financify;User=root;Password=root");
+            optionsBuilder.UseMySql("Server=localhost;port=3306;Database=financify;User=root;Password=root");
```

Comment:
Mantas knisa prota del pakeisto slaptadodzio. wait till he sees this

Reply:
Viskas cia gerai